### ISSUE TITLE
NVDA using 41% or greater of the CPU

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5371]
**Reported by joshknnd1982 on 2015-09-18 16:28**
In windows10 64bit I noticed that whether addons are enabled or disabled NVDA is using a lot of my CPU according to windows10 resource monitor. Please fix this issue so NVDA will not use as much CPU so my and others machines run cooler.

**Comment by bhavyashah** [comment]
On a Windows 8.1 64 bit system, I was recently encountering immense CPU loads due to NVDA in version 2017.1, but nothing alarming with 2017.2. Does the described issue still exist at anyone's end? If nothing significant is reported here, I suppose this ticket should be safely closed.

**Comment by Brian1Gaff** [comment]
Personally apart from an odd spike that is hard to trace, not for me at 
least. Of course its usage can go up when scrolling down a folder etc, and 
if the processor is throttled back as is often the case on multi  core 
machines. Maybe the real culprit is why windows cannot see nvda wants a 
faster clock speed.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by leonardder** [comment]
I've seen CPU usages around 10% quite often, but quite hard to track this down I'm afraid.

**Comment by Adriani90** [comment]
@joshknnd1982 are you still having this issue?

**Comment by Adriani90** [comment]
Actually it depends which programs are you using while running NVDA. If NVDA needs a huge amount of information and it extracts it from the accessibility API, then I can imagine that this causes performance issues. For example in very big MS Word documents, if you enable "pages", "editor revisions" etc. in document formating settings, then you will see that NVDA will use more power than before. But 40% is really significant. I have never seen that. Which programs are you using at the same time?

**Comment by Adriani90** [comment]
do you have any security software installed?

**Comment by Adriani90** [comment]
No updates since years and this issue is too broadly formulated. No chance to track down exactly what's going on without contributions from the initial author or other people having this problem. Thus, if someone still has performance problems, please open a new issuewith exact steps to reproduce.
Closing as abandoned.
